src/material-experimental/mdc-helpers/_mdc-helpers.scss

```diff
@@ -286,26 +321,46 @@ $mat-typography-level-mappings: (
   }
 }
 
-/// Mixin that sets the color of the focus indicators.
+/// Mixin that customizes focus indicators. This mixin may be included
+/// multiple times throughout your app.
 ///
-/// @param {color|map} $theme-or-color
-///   If theme, focus indicators are set to the primary color of the theme. If
-///   color, focus indicators are set to that color.
+/// @param {map} $config
+///   If given, customizes focus indicators with the configuration. The
+///   configuration map accepts border-style, border-color, border-width,
+///   border-radius, and margin. See the SassDoc annotation for
+///   mat-strong-focus-indicator for more on each parameter.
 ///
-/// @example
-///   .demo-dark-theme {
-///     @include mat-mdc-strong-focus-indicators-theme($dark-theme-map);
-///   }
+/// @param {boolean} $cascade
+///   If true, the focus indicator customizations will apply to the
+///   entire element subtree. Otherwise, they only apply to the selector
+///   hosting the mixin. Defaults to true.
 ///
 /// @example
-///   .demo-red-theme {
-///     @include mat-mdc-strong-focus-indicators-theme(#f00);
+///   .my-component {
+///     @include mat-mdc-strong-focus-indicator-customize((
+///       border-radius: 50%,
+///       color: red,
+///     ));
 ///   }
-@mixin mat-mdc-strong-focus-indicators-theme($theme-or-color) {
-  .mat-mdc-focus-indicator::before {
-    border-color: if(
-      type-of($theme-or-color) == 'map',
-      mat-color(map_get($theme-or-color, primary)),
-      $theme-or-color);
+@mixin mat-mdc-strong-focus-indicator-customize($config: (), $cascade: true) {
```

Comment:
We typically default our boolean variables to `false`, maybe rename it to `$no-cascade` or `$prevent-cascade`

Reply:
`$prevent-cascade` SGTM